docs/usage/gitlab-bot-security.md

```diff
@@ -83,6 +83,11 @@ Please see the [renovate-bot/renovate-runner README on GitLab](https://gitlab.co
 We're trying to find a workable design for the GitLab app, so we can enable it safely again.
 If you have any ideas, open a [discussion](https://github.com/renovatebot/renovate/discussions) and let us know!
 
+## Current status
+
+**Update:** GitLab now introduced Group Access Tokens & API for paid & self-hosted instances, but a good permission setup/flow is still not possible.
```

Comment:
```suggestion
GitLab introduced Group Access Tokens & API for paid & self-hosted instances, but a good permission setup/flow is still not possible.
```

Reply:
This is the only suggestion I don't quite agree with, as the rest of the page is outdated and this would clarify that it's a new addition - as the content conflicts somehow with the rest of the page (e.g. `Group Access Tokens are still in planning stage`)

Reply:
Added it anyways - as the heading provides sufficient context